Blockzilla/URLBar.swift

```diff
@@ -582,7 +582,6 @@ class URLBar: UIView {
         updateLockIcon()
         updateUrlIcons()
         let _ = urlText.resignFirstResponder()
-        setTextToURL()
```

Comment:
Hey @oliviabrown9, I've tested this solution with my PR #544 on my local env, and I think here you have to set `setTextToURL(displayFullUrl: false)` or `setTextToURL()`, because when the user click and dismiss the URLBar it doesn't trigger the function and keep the full url and not the truncated one.

Reply:
Good catch :)